apps/api/views/homework.py

```diff
@@ -25,6 +28,15 @@ def perform_update(self, serializer):
         new_obj = serializer.save()
         new_obj.calculate_grade()
 
+class SubmissionGETMultipleView(APIView):
+    def post(self, request, **kwargs):
+        if request.data:
```

Comment:
I think we should probably have some sort of sanity/verification check on the data here before using it in a queryset. Whether that's checking type, or that it the array contains the correct type. Also if not already we should check the permissions on this view. (IsAuthenticated probably makes sense, or Submission perms?

Also we could probably accomplish the same thing on the by overriding the `get_queryset()` method on the existing Submission Serializer by allowing a query param that's a list of PK's to fetch.

Reply:
I planned on using this, but ended up using the templating in Django.